<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>780</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>654</width>
    <height>780</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>654</width>
    <height>780</height>
   </size>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>731</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="btn_order">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>190</y>
        <width>400</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string>주문하시려면 클릭해주세요</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>611</width>
        <height>371</height>
       </rect>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_1">
       <attribute name="title">
        <string>종류 1</string>
       </attribute>
       <widget class="QPushButton" name="menuBtn_1_1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>140</width>
          <height>200</height>
         </rect>
        </property>
        <property name="text">
         <string>메뉴 1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="menuBtn_1_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>70</y>
          <width>140</width>
          <height>200</height>
         </rect>
        </property>
        <property name="text">
         <string>메뉴 2</string>
        </property>
       </widget>
       <widget class="QPushButton" name="menuBtn_1_3">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>70</y>
          <width>140</width>
          <height>200</height>
         </rect>
        </property>
        <property name="text">
         <string>메뉴 3</string>
        </property>
       </widget>
       <widget class="QPushButton" name="menuBtn_1_4">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>70</y>
          <width>140</width>
          <height>200</height>
         </rect>
        </property>
        <property name="text">
         <string>메뉴 4</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>종류 2</string>
       </attribute>
       <widget class="QPushButton" name="menuBtn_2_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>70</y>
          <width>140</width>
          <height>200</height>
         </rect>
        </property>
        <property name="text">
         <string>메뉴 6</string>
        </property>
       </widget>
       <widget class="QPushButton" name="menuBtn_2_4">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>70</y>
          <width>140</width>
          <height>200</height>
         </rect>
        </property>
        <property name="text">
         <string>메뉴 8</string>
        </property>
       </widget>
       <widget class="QPushButton" name="menuBtn_2_1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>140</width>
          <height>200</height>
         </rect>
        </property>
        <property name="text">
         <string>메뉴 5</string>
        </property>
       </widget>
       <widget class="QPushButton" name="menuBtn_2_3">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>70</y>
          <width>140</width>
          <height>200</height>
         </rect>
        </property>
        <property name="text">
         <string>메뉴 7</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_beverage">
       <attribute name="title">
        <string>음료</string>
       </attribute>
       <widget class="QPushButton" name="menuBtn_3_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>70</y>
          <width>140</width>
          <height>200</height>
         </rect>
        </property>
        <property name="text">
         <string>메뉴 10</string>
        </property>
       </widget>
       <widget class="QPushButton" name="menuBtn_3_1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>140</width>
          <height>200</height>
         </rect>
        </property>
        <property name="text">
         <string>메뉴 9</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QTextBrowser" name="text_mname">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>161</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="text_count">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>450</y>
        <width>41</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="text_price">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>450</y>
        <width>111</width>
        <height>192</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_cancel">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>450</y>
        <width>120</width>
        <height>192</height>
       </rect>
      </property>
      <property name="text">
       <string>지우기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_pay">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>450</y>
        <width>120</width>
        <height>192</height>
       </rect>
      </property>
      <property name="text">
       <string>주문</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;주문 목록&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>120</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;합계&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_cancel2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>460</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>뒤로가기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_pay2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>460</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>결제하기</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="text_total">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>200</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>377</width>
        <height>461</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>메뉴</string>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="font">
        <font>
         <kerning>true</kerning>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>수량</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>가격</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>271</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;주문 완료&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="text_finish">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>371</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_back">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>540</y>
        <width>161</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>처음으로 돌아가기</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>654</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
